testing/conftest.py

```diff
@@ -14,3 +14,7 @@ def create_signer(private_key):
     """
     private_key = Secp256k1PrivateKey.from_hex(private_key)
     return CryptoFactory(create_context('secp256k1')).new_signer(private_key)
+
+
+def create_private_key():
```

Comment:
`remme-core/remme/shared/utils.py` we have already a generation for private key here

Reply:
@DeV1doR, done.